docs/admin/high-availability.md

```diff
@@ -188,11 +183,7 @@ them to talk to the external load balancer's IP address.
 So far we have set up state storage, and we have set up the API server, but we haven't run anything that actually modifies
 cluster state, such as the controller manager and scheduler.  To achieve this reliably, we only want to have one actor modifying state at a time, but we want replicated
 instances of these actors, in case a machine dies.  To achieve this, we are going to use a lease-lock in etcd to perform
-master election.  On each of the three apiserver nodes, we run a small utility application named `podmaster`. It's job is to implement a master
-election protocol using etcd "compare and swap". If the apiserver node wins the election, it starts the master component it is managing (e.g. the scheduler), if it
-loses the election, it ensures that any master components running on the node (e.g. the scheduler) are stopped.
-
-In the future, we expect to more tightly integrate this lease-locking into the scheduler and controller-manager binaries directly, as described in the [high availability design proposal](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/proposals/high-availability.md)
+master election.  We will use the `--leader-elect` flag for each scheduler and controller-manager, using etcd locks this will ensure that only 1 instance of the scheduler and controller-manager are running at once.

```

Comment:
I'd change this to say "lease in the API" instead of "etcd locks", the leaderelection client does not connect to etcd